suite-native/biometrics/src/biometricsAtoms.ts

```diff
@@ -25,3 +25,16 @@ export const useIsBiometricsEnabled = () => {
         setIsBiometricsOptionEnabled,
     };
 };
+
+const isBiometricsOverlayVisibleAtom = atom(false);
```

Comment:
Since this atom is set just based on `useIsUserAuthenticated` and `useIsBiometricsEnabled` atoms, It could be derived implicitly as derived read-only atom. Please change, there will be less room for a mistake in using this API. You can delete `useIsBiometricsOverlayVisible` then.

Reply:
Good point. Fixed here https://github.com/trezor/trezor-suite/pull/8647/commits/137dc3dca7d9783427b73039c642466dd0b6eb4c